docs/core/deploying/index.md

```diff
@@ -1,98 +1,169 @@
 
-## Self-contained deployments (SCD)
 
-For a self-contained deployment, you deploy your app and any required third-party dependencies along with the version of .NET Core that you used to build the app. Creating an SCD doesn't include the [native dependencies of .NET Core](https://github.com/dotnet/core/blob/master/Documentation/prereqs.md) on various platforms, so these must be present before the app runs. For more information on version binding at runtime, see the article on [version binding in .NET Core](../versions/selection.md).
+Once you have acquired the tools, you're ready to build your app. Pick the approach that corresponds to whether in the previous step you chose to use an IDE like Visual Studio or Visual Studio for Mac or the CLI.
 
-Starting with NET Core 2.1 SDK (version 2.1.300), .NET Core supports *patch version roll forward*. When you create a self-contained deployment, .NET Core tools automatically include the latest serviced runtime of the .NET Core version that your application targets. (The latest serviced runtime includes security patches and other bug fixes.) The serviced runtime does not have to be present on your build system; it is downloaded automatically from NuGet.org. For more information, including instructions on how to opt out of patch version roll forward, see [Self-contained deployment runtime roll forward](runtime-patch-selection.md).
+[Build your first .NET Core app using Visual Studio](https://docs.microsoft.com/dotnet/core/tutorials/with-visual-studio)
+ 
+[Build your first .NET Core app using the Command Line Interface (CLI)](../tutorials/using-with-xplat-cli.md)
 
-FDD and SCD deployments use separate host executables, so you can sign a host executable for an SCD with your publisher signature.
 
-### Why deploy a self-contained deployment?
+Either choice will allow you to create various types of .NET Core applications including ASP.NET Core, WPF, and Windows Forms applications.
 
-Deploying a Self-contained deployment has two major advantages:
 
-- You have sole control of the version of .NET Core that is deployed with your app. .NET Core can be serviced only by you.
+<br/>
 
-- You can be assured that the target system can run your .NET Core app, since you're providing the version of .NET Core that it will run on.
+### Step 2: Publish your app
 
-It also has a number of disadvantages:
+You can publish your app in one of two configurations - as a [Framework Dependent Deployment (FDD)](publish.md#Framework-Dependent-Deployment-FDD) or as a [Self-Contained Deployment (SCD)](publish.md#Self-contained-deployment-SCD).
 
-- Because .NET Core is included in your deployment package, you must select the target platforms for which you build deployment packages in advance.
+For both of the above configurations, you can publish the app either as a set of loose assemblies for either `xcopy` or package up for deployment, or alternatively as a single file executable.
```

Comment:
Multiple uses of `or` here hurts the readability IMO.

Reply:
Ack.